test/ssl-tests/30-extended-master-secret.cnf.in

```diff
@@ -27,6 +27,7 @@ my @tests_tls1_2 = (
         },
         test   => {
           "ExpectedResult" => "Success",
+          "FIPSversion" => "<=3.1.0",
```

Comment:
What is this version check achieving currently? It seems to skip the test if it doesnt match.. (Wont it always match with <=3.1)

Reply:
Yes, skip if no match.

- 3.1.0 doesn't have mandatory EMS support, so it can run this test.
- 3.1.1 (& later) will have mandatory EMS support, so they can't run it.

Reply:
Maybe put this info back into the commit message?

Reply:
Added.